### DISCUSSION TITLE
neo4j import on mac m1

### DISCUSSION THREAD

**Discussion by Juliantreinta** [amundsen-io/amundsen#1982] [Q&A]
I'm getting this error does someone khowa why? 

ERROR: no matching manifest for linux/arm64/v8 in the manifest list entries

**Comment by tenzo** [comment]
Neo4j pushes images for M1 architecture (arm64) starting from version 4.4: https://hub.docker.com/_/neo4j/tags?page=1
What you find in the current amundsen docker-compose or helm files is old 3.x version.

I'm dealing right now with updating the image of Neo4j to a newer version - I've seen somewhere that versions 4.x should be supported by amundsen (unfortunately I cannot find the source for this).
